packages/cli/src/commands/setup/vite/templates/vite.config.ts.template

```diff
@@ -0,0 +1,13 @@
+import { defineConfig, UserConfig } from 'vite'
+
+import redwood from '@redwoodjs/vite'
+
+/**
+ * https://vitejs.dev/config/
+ * @type {import('vite').UserConfig}
+ */
+const viteConfig: UserConfig = {
+  plugins: [redwood()],
+}
```

Comment:
This leaves us with a bit of type duplication in the case of ts files; wonder if we could clean it up? I'll see if it's this way with other commands too

Reply:
ts-to-js doesn't remove these type comments, so its the only way to guarantee both. Do you see an error in your editor? Mine is fine with both the comment and type

Reply:
Nah it's not about an error just about duplication; not something we need to clean up now